api/src/main/java/edu/cornell/mannlib/vitro/webapp/servlet/setup/RDFFilesLoader.java

```diff
@@ -182,12 +184,16 @@ private static Set<Path> getPaths(String parentDir, String... strings) {
 	private static void readOntologyFileIntoModel(Path p, Model model) {
 		String format = getRdfFormat(p);
 		log.debug("Loading " + p);
+		Model memModel = ModelFactory.createDefaultModel();
```

Comment:
Why you are not using here:
```suggestion
		Model memModel = VitroModelFactory.createModel();
```
If we are using ModelFactory.createDefaultModel() we are not using bulk model, bulk graph, etc., right?

Reply:
Because it wouldn't have any effect on performance, so no reason to use more complicated model.